src/docs/sphinx/quickstart.rst

```diff
@@ -85,102 +79,21 @@ build with that compiler.
 Building Serac's Dependencies via Spack/uberenv
```

Comment:
lines 69-77 would also likely benefit from an update: talks about ubuntu 10 and recommends gcc 10.